lib/IAMClient.d.ts

```diff
@@ -0,0 +1,389 @@
+     * of authorization results
+     * @returns{undefined}
+     */
+    checkPolicies(requestContextParams: {
+        constantParams: any;
+        paramaterize?: any;
+    }, userArn: string, options: {
+        reqUid: string;
+    }, callback: Function): undefined;
+    /**
+     * A getter for account canonical IDs given a list of account IDs
+     * @param{String[]} userIds - list of account IDs
+     * @param {Object} options - additional arguments
+     * @param {string} options.reqUid - the request UID
+     * @param {werelogs.RequestLogger} [options.logger] - Logger instance
+     * @param{Function} callback - the callback handling the array of objects
+     * and the error, if there is one
+     * @returns{undefined}
+     */
+    getUsersById(userIds: string[], options: {
+        reqUid: string;
+        logger?: werelogs.RequestLogger;
+    }, callback: Function): undefined;
+    healthcheck(reqUid: any, callback: any): void;
+    /**
+     * @param {string} method - CRUD method chosen for the request
+     * @param {string} path - RESTful URL for the request
+     * @param {boolean} iamAuthenticate - whether to add iam authentication
+     * headers to request
+     * @param {VaultClient~requestCallback} callback - callback
+     * @param {object} data - object containing data to be sent through the req
+     *                        (while metadata is sent in the URL); data may or
+     *                        may not be present depending on the type of
+     *                        request
+     * @param {string} [reqUid] - Request logger uid, to trace request
+     * @param {string} [contentType] - content type of body
+     * @returns {undefined}
+     */
+    request(method: string, path: string, iamAuthenticate: boolean, callback: any, data: object, reqUid?: string, contentType?: string): undefined;
+    /**
+     * This function will parse the json result
+     *
+     * @param {string} ret - Result from request
+     * @return {object|Error} Object parse or the error
+     */
+    parseObj(ret: string): object | Error;
+    /**
+     * This function will return an object from the request result
+     *
+     * @param {string} ret - Result from request
+     * @param {function} cb - Callback(err, obj)
+     * @return {undefined}
+     */
+    getObj(ret: string, cb: Function): undefined;
+    /**
+     * @param {object} res - response to the request
+     * @param {string} ret - content (data and/or metadata) returned after the
+     *                       request has been processed
+     * @param {object} log - werelogs request logger
+     * @param {VaultClient~requestCallback} cb - callback
+     * @returns {undefined}
+     */
+    handleResponse(res: object, ret: string, log: object, cb: any): undefined;
+}
+import { http as HttpAgent } from "httpagent";
+import { https as HttpsAgent } from "httpagent";
+//# sourceMappingURL=IAMClient.d.ts.map
```

Comment:
can we delete this comment line or do we have some specific reason to keep it here?

Reply:
honestly no idea as this was a generated file!